pyproject.toml

```diff
@@ -42,6 +45,9 @@ pylint-django = "*"
 pydocstyle = "*"
 flake8 = "*"
 parameterized = "*"
+python-dotenv = "^0.19.1"
+mysqlclient = "^2.0.3"
```

Comment:
These should be `"*"` versions, especially `mysqlclient` so that it won't conflict when Nautobot changes its dependency version.